lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -336,6 +350,86 @@ class CategoryRenderer {
   }
 
   /**
+   * Render a screenshot filmstrip and metrics bars
+   * @param {!Array<!ReportRenderer.AuditJSON>} metricAudits
+   * @param {*} artifacts
+   * @return {?Element}
+   */
+  _renderPerfTimeline(metricAudits, artifacts) {
+    if (!artifacts)
+      return;
+    const tracePass = artifacts['traces'] ? artifacts['traces']['defaultPass'] : null;
+    if (!tracePass)
+      return;
+
+    const fmp = metricAudits.find(a => a.id === 'first-meaningful-paint').result;
+    if (!fmp || !fmp['extendedInfo'])
+      return;
+
+    const tti = metricAudits.find(a => a.id === 'time-to-interactive').result;
+    if (!tti || !tti['extendedInfo'])
+      return;
+
+    const navStart = fmp['extendedInfo']['value']['timestamps']['navStart'];
+    const markers = [
```

Comment:
it'd be lovely to use [`metricsDefinitions()` in pwmetrics-events.js](https://github.com/GoogleChrome/lighthouse/blob/5fa684ef993f13f9028a306fdfb5e7566c27f7fc/lighthouse-core/lib/traces/pwmetrics-events.js#L47) for this section here, however we're currently in the standalone report.